.github/workflows/ci.yml

```diff
@@ -11,8 +11,8 @@ jobs:
   determine_docker_org_and_tag:
     runs-on: ubuntu-latest
     outputs:
-      docker_organization: ${{ steps.docker-org-and-tag.outputs.docker_organization }}
-      docker_image_tag: ${{ steps.docker-org-and-tag.outputs.docker_image_tag }}
+      docker_organization: usdotfhwastolcandidate
+      docker_image_tag: prowler
```

Comment:
Not necessary since they are already being detected correctly:
![image](https://github.com/user-attachments/assets/5b797fc5-89cb-4ef3-990d-99fe9b51781f)

Reply:
reverted